note-app/src/components/NoteList/NoteList.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import NoteItem from './NoteItem/NoteItem';
+import PropTypes from 'prop-types'
+
+function NoteList({ style, notes, onSelect }) {
+  return (
+    <div style={style}>
+      {notes.map((note, index) => {
+        return <NoteItem note={note} key={index} onSelect={onSelect}></NoteItem>;
+      })}
+    </div>
+  );
+}
```

Comment:
1. Arrow function
2. Remove `return` statement
3. Remove inside the `map` unexpected `return`
4. Empty tag usage
```suggestion
const NoteList = ({ style, notes, onSelect }) => (
    <div style={style}>
      {notes.map((note, index) => (
         <NoteItem note={note} key={index} onSelect={onSelect} />
      ))}
    </div>
  );
```